index.d.ts

```diff
@@ -204,3 +204,23 @@ export type RequireAtLeastOne<ObjectType, KeysType extends keyof ObjectType = ke
 	}[KeysType]
 	// …then, make intersection types by adding the remaining properties to each mapped type.
 	& Omit<ObjectType, KeysType>;
+
+/*
+Converts a readonly object into a mutable object. Inverse of Readonly<T>.
+
```

Comment:
Would be good to include some text on real-world use-cases of this type, with links if possible.